department-of-veterans-affairs/va.gov-cms
# Vagov 4635 disability migration

To test:
1. Go to /admin/structure/migrate/manage/va_new_benefits/migrations/va_benefits_disability/execute and execute migration (import).
2. Go to /admin/structure/migrate/manage/va_new_benefits/migrations and confirm that the line, "Migrate Disability - non-eligibility pages from va.gov" shows 20 migrated out of 21 (one page, https://www.va.gov/disability/about-disability-ratings-beta/, is not migrated because it doesn't exist on the live site).
